.idea/misc.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (DjangoBlog)" project-jdk-type="Python SDK" />
+  <component name="PyCharmProfessionalAdvertiser">
```

Comment:
ignore the .idea folder = add it to .gitignore then delete it from the git version control cache with `git rm -r .idea`

Reply:
Ok, I can do that! :) Is that the next video in the Django tutorial?

Reply:
no, this is related to source control and what to ignore, the .idea folder is related to your PyCharm / your IDE so you would not commit that to version control / GitHub